jbpm-designer-backend/src/main/java/org/jbpm/designer/server/EditorHandler.java

```diff
@@ -85,6 +85,8 @@
      */
     public static final String USEOLDDATAASSIGNMENTS = "designer.useolddataassignments";
 
+    public static final String SHOW_PDF_DOC = "designer.showpdfdoc";
```

Comment:
is this the standard for property naming that you use for designer? I see below org.jbpm.designer.XXX and above designer.XXX. Ideally all properties should follow the same convention.